R/t_summarytable.R

```diff
@@ -2,16 +2,16 @@
 #'
 #' Output descriptive summary statistics table as a data frame. Includes biomarker, treatment,
 #' visit,
-#' n, mean, median, sd, min, max, %missing values, % LOQ values.
+#' n, mean, median, SD, min, max, %missing values, % `LOQ` values.
```

Comment:
Is capitalization of "sd" done solely because the linter accepts "SD" and not "sd"?

Reply:
@chlebowa this is a statistical term and it has official capitalized short-form. You can see a lot of statistical terms being written in this manner, like AIC, PCA, PCR, MLE. And yes, those are omitted by `spelling::spell_check_package`
```{R}
hunspell::en_stats
[1] "Accessor"           "accessor"           "accessors"          "ACF"                "Affymetrix"        
  [6] "agglomerative"      "AIC"                "Akaike"             "Akaike's"           "alikes"            
 [11] "ANOVA"              "API"                "approximative"      "ARIMA"              "ARMA"              
 [16] "ascii"              "AUC"                "autocorrelation"    "autocorrelations"   "autocovariance"    
 [21] "autocovariances"    "autoregression"     "autoregressive"     "autosomal"          "backend"           
 [26] "backfitting"        "backquote"          "backquoted"         "backreference"      "backticks"         
 [31] "barplot"            "barplots"           "basename"           "BFGS"               "BibTeX"            
 [36] "bilinear"           "binomially"         "Bioconductor"       "biplot"             "biplots"           
 [41] "bitwise"            "Bivariate"          "bivariate"          "biweight"           "boolean"           
 [46] "boxplot"            "boxplots"           "breakpoint"         "capita"             "cartesian"         
 [51] "CCA"                "CDF"                "CDFs"               "checkbox"           "Cholesky"          
 [56] "CIs"                "CJK"                "clusterings"        "CMYK"               "codepage"          
 [61] "codings"            "collinearity"       "colnames"           "columnwise"         "componentwise"     
 [66] "cophenetic"         "correlation"        "covariables"        "covariance"         "covariances"       
 [71] "Covariate"          "covariate"          "Covariates"         "covariates"         "Cp"                
 [76] "CPUs"               "CRAN"               "criterium"          "CRLF"               "CSS"               
 [81] "CSV"                "cumulated"          "cutpoint"           "cutpoints"          "datafile"          
 [86] "Dataframe"          "dataframe"          "dataframes"         "datapoints"         "Dataset"           
 [91] "dataset"            "Datasets"           "datasets"           "DBI"                "DCF"               
 [96] "dendrogram"         "dendrograms"        "deparse"            "deparsed"           "deparses"          
[101] "deparsing"          "detrended"          "devel"              "deviances"          "DIC"               
[106] "dichotomized"       "differenced"        "differentiable"     "differentially"     "dimensionality"    
[111] "dimnames"           "discretization"     "discretize"         "discretized"        "DLL"               
[116] "DLLs"               "EDF"                "eigenvector"        "eigenvectors"       "EISPACK"           
[121] "elementwise"        "encodings"          "endian"             "EPS"                "equispaced"        
[126] "ESRI"               "Euclidian"          "exceedance"         "executables"        "expectation"       
[131] "exponentiated"      "FFT"                "Filename"           "filename"           "filenames"         
[136] "filepaths"          "FIPS"               "forecasted"         "Fortran"            "FreeBSD"           
[141] "frequentist"        "functionalities"    "functionals"        "GAM"                "GARCH"             
[146] "gaussian"           "Gaussians"          "GCV"                "genomic"            "genotype"          
[151] "genotyped"          "genotypes"          "genotypic"          "genotyping"         "Ghostscript"       
[156] "gif"                "GIS"                "GLM"                "GLMs"               "Google"            
[161] "GPD"                "GPL"                "grob"               "grobs"              "GUIs"              
[166] "haplotype"          "haplotypes"         "heatmap"            "Helvetica"          "heteroscedastic"   
[171] "heteroscedasticity" "hoc"                "homogenous"         "homoscedastic"      "HPD"               
[176] "HSV"                "html"               "HWE"                "hypergeometric"     "hyperparameter"    
[181] "hyperparameters"    "identifiability"    "IEEE"               "iff"                "iid"               
[186] "individual's"       "informations"       "inline"             "internet"           "interquartile"     
[191] "invariance"         "invertibility"      "IQR"                "IRT"                "IWLS"              
[196] "Jacobian"           "JASA"               "jittered"           "jittering"          "JPEG"              
[201] "Kolmogorov"         "kriging"            "Kruskal"            "Kurtosis"           "kurtosis"          
[206] "LAPACK"             "LaTeX"              "latin"              "LDA"                "LGPL"              
[211] "LHS"                "lhs"                "linearization"      "LINPACK"            "localhost"         
[216] "locator"            "logicals"           "logit"              "logits"             "loglikelihood"     
[221] "loglinear"          "lognormal"          "lookup"             "lowess"             "LRT"               
[226] "Mahalanobis"        "manhattan"          "MANOVA"             "MATLAB"             "Matlab"            
[231] "matrixes"           "maxima"             "MCD"                "MCMC"               "MDI"               
[236] "medoids"            "metacharacters"     "metadata"           "microarray"         "microarrays"       
[241] "microsatellite"     "minima"             "minimax"            "misclassification"  "missingness"       
[246] "missings"           "MLE"                "MLEs"               "monospaced"         "MPI"               
[251] "MSE"                "multi"              "multibyte"          "multicore"          "multiline"         
[256] "multilocus"         "multimodal"         "Multinomial"        "multinomial"        "multinomially"     
[261] "namespace"          "namespaces"         "NAs"                "NCBI"               "nonlinearity"      
[266] "nonnegative"        "Nonparametric"      "nonparametric"      "numerics"           "OECD"              
[271] "ok"                 "OLS"                "Omegahat"           "one's"              "online"            
[276] "orthogonalized"     "outlier"            "outliers"           "overdispersed"      "overdispersion"    
[281] "overfitting"        "overlayed"          "overplotting"       "pairlist"           "pairlists"         
[286] "parallelized"       "parameterisation"   "parameterization"   "parameterizations"  "parameterize"      
[291] "parameterized"      "parametrization"    "parametrizations"   "pathname"           "patient's"         
[296] "PCA"                "PCR"                "PCRE"               "PDF"                "pdf"               
[301] "periodogram"        "phenotypic"         "Phylogenetic"       "phylogenetic"       "piecewise"         
[306] "PLS"                "plugin"             "PNG"                "pointsize"          "pointwise"         
[311] "polytomous"         "POSIX"              "PostScript"         "precalculated"      "prepend"           
[316] "prepended"          "preprocessing"      "prespecified"       "probit"             "pseudomedian"      
[321] "QQ"                 "QTL"                "Quantile"           "quantile"           "Quantiles"         
[326] "quantiles"          "randomizations"     "rbind"              "reals"              "regressor"         
[331] "regressors"         "REML"               "reparameterized"    "resample"           "resampled"         
[336] "resamples"          "Resampling"         "resampling"         "resize"             "resized"           
[341] "reweighted"         "RGB"                "rhs"                "RMSE"               "RNG"               
[346] "ROC"                "rownames"           "RSS"                "runtime"            "SANN"              
[351] "SAS"                "scalings"           "Scatterplot"        "scatterplot"        "scatterplots"      
[356] "screeplot"          "semidefinite"       "semiparametric"     "shapefile"          "sib"               
[361] "significand"        "skewness"           "Smirnov"            "smoothers"          "SNP"               
[366] "SNPs"               "Solaris"            "Spearman"           "Spearman's"         "spinogram"         
[371] "spinograms"         "Splus"              "SPSS"               "SQL"                "sqrt"              
[376] "startup"            "stationarity"       "stepsize"           "stepwise"           "stochastically"    
[381] "studentized"        "subdirectories"     "subdirectory"       "subexpression"      "subexpressions"    
[386] "subgraph"           "subintervals"       "sublist"            "sublists"           "submatrices"       
[391] "submatrix"          "submodel"           "submodels"          "subpopulations"     "subsample"         
[396] "subsamples"         "subsampling"        "subsequence"        "subseries"          "subsetted"         
[401] "Subsetting"         "subsetting"         "substring"          "substrings"         "subtree"           
[406] "subtrees"           "superclass"         "superclasses"       "superset"           "supremum"          
[411] "SVD"                "SVG"                "SVM"                "Sweave"             "taxa"              
[416] "taxon"              "Tcl"                "TCP"                "TeX's"              "thresholded"       
[421] "thresholding"       "tickmarks"          "timepoints"         "timestamp"          "timestep"          
[426] "Tk"                 "TODO"               "tradeoff"           "Tukey"              "Tukey's"           
[431] "tuple"              "txt"                "unary"              "unexported"         "unimodal"          
[436] "uniquenesses"       "univariate"         "unix"               "unlisting"          "unnormalized"      
[441] "unpenalized"        "unstandardized"     "url"                "URW"                "USD"               
[446] "username"           "UTC"                "UTF"                "utils"              "UTM"               
[451] "variable's"         "variance"           "variogram"          "vertices"           "viewport"          
[456] "viewport's"         "viewports"          "voxel"              "webpage"            "Weibull"           
[461] "whitespace"         "Wilcoxon"           "wildcard"           "wildcards"          "wireframe"         
[466] "Wishart"            "wrt"                "x's"                "xml"
```